master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/ha/HARaftServer.java

```diff
@@ -375,6 +386,37 @@ private RaftProperties newRaftProperties(CelebornConf conf) {
     return properties;
   }
 
+  private Parameters configureSsl(CelebornConf conf, RaftProperties properties, RpcType rpc) {
+
+    if (rpc != SupportedRpcType.GRPC) {
+      LOG.error(
+          "SSL has been disabled for Raft communication between masters. "
+              + "This is only supported when ratis is configured with GRPC");
```

Comment:
QQ: The default is NETTY - are there any concerns with changing default to GRPC ?

Reply:
If a rolling upgrade of the master is performed, might there be compatibility issues with Netty RPC here?

Reply:
That is a good point, agree - thanks !
Let us leave it as is, I have documented it explicitly to use GRPC until Ratis (and so we) add support for netty.